webpack.config.js

```diff
@@ -127,7 +127,12 @@ var wpconfig = {
     },
 
     externals: {
-        gettext: 'gettext'
+        backbone: 'Backbone',
```

Comment:
Why doesn't this contain the complete list of external libraries (https://github.com/edx/edx-platform/pull/15301/files#diff-1bc8d56d7f3ab9bffca59e6e0b1c030dR65)?